packages/widgetbook/lib/src/fields/field_codec.dart

```diff
@@ -32,18 +32,33 @@ class FieldCodec<T> {
     return '{${pairs.join(',')}}';
   }
 
+  /// Tring to decode [component] if possible.
+  ///
+  /// If not possible returned null.
+  static String? tryDecodeComponent(String component) {
+    try {
+      return Uri.decodeComponent(component);
+    } on ArgumentError {
+      return null;
+    }
+  }
+
   /// Decodes a query group encoded value back to a [Map].
   static Map<String, String> decodeQueryGroup(String? group) {
     if (group == null) return {};
 
     final params = group.substring(1, group.length - 1).split(',');
```

Comment:
But IMHO in textfields frequently use symbol `,`, maybe replace this symbol on `|` or `;`.

Reply:
This is a good suggestion, we will consider it for the next major release. But for now, it might be breaking some link for some users if the links still use the old format.